packages/bpk-mixins/_typography.scss

```diff
@@ -498,6 +498,49 @@
   );
 }
 
+/// Editorial 1 text style
+///
+/// @example scss
+///   .selector {
+///     @include bpk-editorial-1;
+///   }
+
+@mixin bpk-editorial-1 {
+  font-family: $bpk-font-family-larken;
+
+  @include _bpk-text-factory($bpk-font-size-xxxxl, $bpk-line-height-xxxxl, 300);
```

Comment:
[The Figma has the Editorial 1 and Editorial 2 texts with a font-weight of 300](https://www.figma.com/file/KXf2gHNLDe2cXWUoHl4cTX/branch/n9kEhgih1ClQu3Ah5CgSkQ/Backpack-Foundations?type=design&node-id=3771-4918&mode=design&t=HOun2KGHC3WBIdNs-0), but this `font-weight` doesn't exist as a token in `bpk-foundations`.

![image](https://github.com/Skyscanner/backpack/assets/19427244/943cf6b3-1fc5-4761-93b0-61e9cb5a01b4)

Should this also be added in as new `$bpk-font-weight-light`?

Reply:
I think that is a good idea for us to add a new weight token for this

Reply:
Added as new token here https://github.com/Skyscanner/backpack-foundations/pull/313